tests/e2e-openshift-upgrade/upgrade/upgrade-operator.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: tempo-product
```

Comment:
does it upgrade only tempo?

Reply:
The tempo upgrade is not required as we wont have the FBC fragment image in OTEL FBC test. Updated to upgrade OTEL only.